yksmp/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "yksmp"
+version = "0.1.0"
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib"]
```

Comment:
This will cause linkage problems later down the line. From memory, if you have `>1` cdylib crate linked, then you get multiple copies of the Rust std lib and they symbols clash.

Can it be a rlib instead?

Reply:
Actually might not be needed at all anymore now that I've moved it into yk. This used to be a standalone repo.

Reply:
Fixed in 0f86fdd.

Reply:
Do the tests still pass? I see that you are exposing ` __yk_stopgap()` to C.

This symbol should probably go in `ykcapi`. You can still write it in Rust by having ` __yk_stopgap()` as a stub which calls into `yksmap`.

Reply:
If we do this, the symbol name for the wrapper should probably be `__yksmap_stopgap()`. That's the convention we've used so far.

Reply:
This isn't hooked properly into yk yet, so this function isn't really used anywhere atm. I think these changes should probably go into the new PR that hooks everything in.

Reply:
> Do the tests still pass?

I see, no test yet calls `__yk_stopgap`.

Reply:
I could possibly test this by calling `__yk_stopgap` from `__llvm_deoptimize` in the `deopt.c` test, but then we would have to add langtester to this. Since I was going to use langtester once this is hooked into yk, I felt there is no need to do it here as well.

Reply:
Perhaps we should add an `unreachable!()` to `__yk_stopgap`? It feels weird to have code that's not being tested at all.